.NET/Microsoft.Recognizers.Text.DataDrivenTests/Number/TestNumber_Swedish.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.Recognizers.Text.DataDrivenTests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Recognizers.Text.Number.Tests
+{
+    [TestClass]
+    public class TestNumber_Swedish : TestBase
```

Comment:
Let's remove these Number test skeleton classes from this PR. Build is failing as they don't currently run.

Reply:
```
Failed   NumberModel
Error Message:
 Class Initialization method Microsoft.Recognizers.Text.Number.Tests.TestNumber_Swedish.ClassInitialize threw exception. System.IO.DirectoryNotFoundException: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\projects\recognizers-text\Specs\Number\Swedish'..
Stack Trace:
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileSystemEnumerableIterator`1.CommonInit()
   at System.IO.FileSystemEnumerableIterator`1..ctor(String path, String originalUserPath, String searchPattern, SearchOption searchOption, SearchResultHandler`1 resultHandler, Boolean checkHost)
   at System.IO.Directory.GetFiles(String path, String searchPattern)
   at Microsoft.Recognizers.Text.DataDrivenTests.TestResourcesExtensions.InitFromTestContext(TestResources resources, TestContext context) in C:\projects\recognizers-text\.NET\Microsoft.Recognizers.Text.DataDrivenTests\TestResourcesExtensions.cs:line 18
   at Microsoft.Recognizers.Text.Number.Tests.TestNumber_Swedish.ClassInitialize(TestContext context) in C:\projects\recognizers-text\.NET\Microsoft.Recognizers.Text.DataDrivenTests\Number\TestNumber_Swedish.cs:line 14
```

Reply:
ok

Reply:
Fixed